src/App.js

```diff
@@ -1,22 +1,28 @@
-import React, { useState } from 'react';
-import calculate from './logic/calculate';
-import Calculator from './components/Calculator';
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import RouteComponent from './components/RouteComponent';
 
-const App = () => {
-  const [state, setState] = useState({
-    total: 0,
-    next: 0,
-  });
-
-  const clickHandler = (e) => {
-    setState((prev) => calculate(prev, e.target.value));
-  };
-
-  return (
-    <>
-      <Calculator handleClick={clickHandler} calcState={state} />
-    </>
-  );
-};
+const App = () => (
+  <React.StrictMode>
+    <BrowserRouter>
+      <RouteComponent />
+    </BrowserRouter>
+  </React.StrictMode>
+);
 
 export default App;
+// import React, { Component } from 'react';
+// import { BrowserRouter } from 'react-router-dom';
+// import RouteComponent from './components/RouteComponent';
+
+// export default class App extends Component {
+//   render() {
+//     return (
+//       <React.StrictMode>
+//         <BrowserRouter>
+//           <RouteComponent />
+//         </BrowserRouter>
+//       </React.StrictMode>
+//     );
+//   }
+// }
```

Comment:
- [ ] Please remove commented pieces of code as it makes your codebase look messy and unprofessional. You can use git history if you need to look into a previous version of your code.